ironic/tests/unit/drivers/modules/drac/test_management.py

```diff
@@ -532,3 +532,31 @@ def test_get_sensors_data(self, mock_get_drac_client):
                                   shared=False) as task:
             self.assertRaises(NotImplementedError,
                               task.driver.management.get_sensors_data, task)
+
+    def test_reset_idrac(self, mock_get_drac_client):
+        mock_client = mock.Mock()
+        mock_get_drac_client.return_value = mock_client
+
+        with task_manager.acquire(self.context, self.node.uuid,
+                                  shared=False) as task:
+            return_value = task.driver.management.reset_idrac(task)
+            mock_client.reset_idrac.assert_called_once_with(
+                force=True, wait=True)
+
+            self.assertIsNone(return_value)
+            self.node.refresh()
```

Comment:
The node.refresh() line isn't needed above.  It causes the node to be reloaded from the DB, which is only needed before doint asserts on properties of the node.

Reply:
Done